src/test/java/uk/gov/justice/log/integration/SearchServiceIT.java

```diff
@@ -0,0 +1,500 @@
+package uk.gov.justice.log.integration;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.isJson;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.withJsonPath;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static uk.gov.justice.common.TestMockDataFiles.mockSetupForSearchCriteria;
+
+import uk.gov.justice.common.AbstractIntegrationTest;
+import uk.gov.justice.log.search.ElasticSearchQueryBuilder;
+import uk.gov.justice.log.search.SearchService;
+import uk.gov.justice.log.utils.PropertyReader;
+import uk.gov.justice.log.utils.SearchConfig;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import org.apache.http.util.EntityUtils;
+import org.elasticsearch.client.Response;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
They should run independently so removed, used when testing locally for easy visualisation of the results